1-mandatory/2-people-i-know.js

```diff
@@ -393,15 +395,34 @@ You can see who people's colleagues are by seeing the "colleagues" array in each
 This time, I only want the full names ("<firstname> <lastname>") of my friends who are colleagues of hers.
 */
 
-let friendsWhoAreColleaguesOfStacie = [];
+let friendsWhoAreColleaguesOfStacie = friends
+  .filter((element) => {
+    for (let el of element.colleagues) {
```

Comment:
amazing approach but it's hard to read, it's better to use 'for' loop instead of javascript methods to make the code more readable.

Reply:
Thank you @ali-nasir-ali! I truly appreciate your review and I will try to use 'for loop' on VS code to see if I can generate the same results. Thanks